SCDoc/SCDoc.y

```diff
@@ -183,15 +183,15 @@ optsubsubsections: subsubsections
 subsubsections: subsubsections subsubsection { $$ = doc_node_add_child($1,$2); }
               | subsubsection { $$ = doc_node_make("(SUBSUBSECTIONS)",NULL,$1); }
               | body { $$ = doc_node_make_take_children("(SUBSUBSECTIONS)",NULL,$1); }
-; 
+;
 
-subsubsection: METHOD methnames optMETHODARGS eol methodbody
-    {
-        $2->id = "METHODNAMES";
-        $$ = doc_node_make(method_type,$3,$2);
-        doc_node_add_child($$, $5);
-//        doc_node_add_child($2, $3);
-    }
+subsubsection: SUBSUBSECTION words2 eol optbody { $$ = doc_node_make_take_children("SUBSUBSECTION", $2, $4); }
+             | METHOD methnames optMETHODARGS eol methodbody
+               {
+                 $2->id = "METHODNAMES";
+                 $$ = doc_node_make(method_type,$3,$2);
+                 doc_node_add_child($$, $5);
+               }
```

Comment:
I think we don't need to touch these lines of code? So this could be sufficient?

```suggestion
    {
        $2->id = "METHODNAMES";
        $$ = doc_node_make(method_type,$3,$2);
        doc_node_add_child($$, $5);
//        doc_node_add_child($2, $3);
    }
```